kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkTaskConfig.java

```diff
@@ -236,6 +259,11 @@ private void checkPartitionConfigs() {
               + "Use either bigQueryPartitionDecorator OR timestampPartitionFieldName."
       );
     }
+    getPartitionExpirationMs().ifPresent(partitionExpiration -> {
+      if (partitionExpiration <= 0) {
+        throw new ConfigException("The partition expiration value must be positive.");
```

Comment:
`ConfigException` has a neat constructor that also includes information on the user-specified key and value; we should probably use that here:
```suggestion
        throw new ConfigException(BIGQUERY_PARTITION_EXPIRATION_CONFIG, partitionExpiration, "The partition expiration value must be positive.");
```

Reply:
Nice!